Samples/iOS-Swift/iOS-Swift.xcodeproj/project.pbxproj

```diff
@@ -715,6 +735,10 @@
 			dependencies = (
 			);
 			name = "iOS13-Swift";
+			packageProductDependencies = (
+				84BA71F42C8F74660045B828 /* SwiftMessages */,
+				84BA71F62C8F74660045B828 /* Anchorage */,
+			);
```

Comment:
Heh, it's biting me right now with the iOS <13 build failures. I added them just since I've used them for so long and they are stable libs. But true, we don't really need them, I can accomplish the task without them.